lib/watir/rspec/cli.rb

```diff
@@ -88,6 +88,8 @@ def install
 end
   ]
           end
+
+          puts "WARNING : the HTML formatter cannot be used until the rspec legacy formatter gem is also installed !! (https://github.com/rspec/rspec-legacy_formatters)"
```

Comment:
Since we are already forcing to use `RSpec 3` then why not include that legacy formatter gem as a `watir-rspec` dependency as long as formatter hasn't updated to `RSpec 3` as well? Do you see any problems by doing like that?